lib/daru/dataframe.rb

```diff
@@ -2282,8 +2274,8 @@ def prepare_sort_opts vector_order, opts
         by: {}
       }.merge(opts)
 
-      opts[:ascending]   = coerce_sort_order vector_order, opts[:ascending]
-      opts[:handle_nils] = coerce_handle_nils vector_order, opts[:handle_nils]
+      opts[:ascending]   = coerce_boolean_array opts[:ascending], vector_order.size, ':ascending'
```

Comment:
Ah, ok :) Went through set of changes here, just forgot the cleanup after last change. Though, current code looks like it WANTS to be simplified into something like `coerce_boolean_array opts, :ascending, vector_order.size` (and sets/gets the key inside).